src/data/Web Development/bugzilla.yaml

```diff
@@ -0,0 +1,10 @@
+name: Bugzilla
+description: |
+  ### Extra information for Bugzilla
+
+  
+
+  #### Get involved with the [Bugzilla](https://wiki.mozilla.org/Bugzilla) project or [BMO](https://wiki.mozilla.org/BMO), Mozilla's custom version
```

Comment:
Could we remove the `####`. As a result, I think the markdown component will wrap it in a  `p` tag by default.

```
### Extra information for Bugzilla

Get involved with the [Bugzilla](https://wiki.mozilla.org/Bugzilla) project or [BMO](https://wiki.mozilla.org/BMO), Mozilla's custom version
```

If you could do the same thing for the other `yaml` files, that would be awesome :)

Reply:
@helfi92 Sure, I modified it. Please check :D

Reply:
@fiennyangeln Any reason why 3 new lines between `###` and the description? Why not 1?

Reply:
It is because the library that I use html2text add new lines in between the tag and initially I think it has to be that way.. Changed :D thanks for the catch